website/templates/legals.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
```

Comment:
This isn't exactly needed...
(This would a folder in the repo with markdown files)

Reply:
So what are we going to add to the website ?